exporting/pubsub/README.md

```diff
@@ -0,0 +1,34 @@
+<!--
+---
+title: "Export metrics to Google Cloud Pub/Sub Service"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/exporting/pubsub/README.md
+---
+-->
+
+# Export metrics to Google Cloud Pub/Sub Service
+
+## Prerequisites
+
+To use the Pub/Sub service for metric collecting and processing, you should first
+[install](https://github.com/googleapis/cpp-cmakefiles) Google Cloud Platform C++ Proto Libraries.
+Pub/Sub support is also dependent on the dependencies of those libraries, like `protobuf` and `grpc`. Next, Netdata
+should be re-installed from the source. The installer will detect that the required libraries are now available.
+
+## Configuration
+
+To enable data sending to the Pub/Sub service, run `./edit-config exporting.conf` in the Netdata configuration directory
+and set the following options:
+
+```conf
+[pubsub:my_instance]
+    enabled = yes
+    destination = pubsub.googleapis.com
+    credentials file = /etc/netdata/google_cloud_credentials.json
+    project id = my_project
+    topic id = my_topic
+```
+
+Set the `destination` option to a Pub/Sub service endpoint. `pubsub.googleapis.com` is the default one.
+You should provide access to the file with Google Cloud credentials for the `netdata` user.
```

Comment:
Is this saying that the `netdata` user should be able to read the `google_cloud_credential.json` file? Maybe there's another way to phrase that.

And where does `google_cloud_credential.json` come from? Do they download the file and put it in `/etc/netdata`, or create it on their own?

Reply:
Yes and yes.
As for the downloading, please look ad https://cloud.google.com/docs/authentication/getting-started. Should we add this link to the documentation?